tests/tests.py

```diff
@@ -56,6 +74,22 @@ def create_webpage(filename, before, after):
             f_html.write(svg)
             f_html.write("</body></html>")
 
+    def _are_identical(self, reference_file: str, new_file: str) -> bool:
+        if (reference_file.endswith(".svg") or reference_file.endswith(".png")) and (
+            new_file.endswith(".png") or new_file.endswith(".svg")
+        ):
+            reference_png, new_png = _get_png(reference_file), _get_png(new_file)
+
+            reference_image = Image.open(reference_png.absolute()).convert("RGB")
+            new_image = Image.open(new_png.absolute()).convert("RGB")
+            diff = ImageChops.difference(reference_image, new_image)
+
+            return not bool(
+                diff.getbbox()
+            )  # if diff.getbbox():  #     return False  # else:  #     return True
```

Comment:
Is this commented out code useful or was it left by mistake?

Reply:
Will fix in a new PR, when work with the updated base image is completed